functions/dataManger/collectionManager.ts

```diff
@@ -0,0 +1,71 @@
+const AWS = require('aws-sdk');
+const isTest = process.env.JEST_WORKER_ID;
```

Comment:
I feel like we could aggregate all this logic for creating the DDB worker and determining to configure it with test settings or not in some central module. That way if you ever wanted to change it, you'd only have to do it in one place.